### ISSUE TITLE
Sometimes on kill process simplewallet corrupts wallet file

### ISSUE THREAD

**Issue by pmitchev** [forknote/forknote#30]
ERROR   Wallet initialize failed: can't load wallet file '/home/fork/forknote-linux/wallets/dashcoin.pool.wallet', check password

**Comment by cycleworm** [comment]
can the wallet be restored with importing feature?

**Comment by pmitchev** [comment]
yes. Use simplewallet with "i" option.  But you have to have the private keys exported first (export_keys in simplewallet).
If your wallet file is corrupted and you don't have the keys there is a .backup file created in the same directory you can use to restore

**Comment by cycleworm** [comment]
ah thx for the quick reply. i tried it an got spend and view secrect key.
are there any other features of simplewallet which are not listed when typing --help ?

where are things like "export_keys" documented?

**Comment by pmitchev** [comment]
I just saw it's not in the site (http://forknote.net), but you can see it with "help"

**Comment by cycleworm** [comment]
ah yeah. it is when you type help when simplewallet is running. 
not simplewallet --help. 

are you the host of (http://forknote.net) ?

**Comment by pmitchev** [comment]
Yes, if you see something you can add you can make pull request anytime

**Comment by pmitchev** [comment]
This issue still exists. If someone expirience it - when the .wallet file corrupts, simplewallet creates .tmp file. Replace the wallet file with the .tmp file to get it fixed.
